lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -78,6 +82,13 @@ describe('asset-saver helper', () => {
       fs.unlinkSync(traceFilename);
     });
 
+    it('devtools log file saved to disk with data', () => {
+      const filename = 'the_file-0.devtools.json';
+      const fileContents = fs.readFileSync(filename, 'utf8');
+      assert.ok(fileContents.includes('"message": "first"'));
```

Comment:
are there any constraints around the format for this file?  (aka not pretty print?)